src/core/core_utils.js

```diff
@@ -105,10 +106,6 @@ function getInheritableProperty({
       }
       values.push(value);
     }
-    if (++loopCount > LOOP_LIMIT) {
-      warn(`getInheritableProperty: maximum loop count exceeded for "${key}"`);
-      break;
-    }
```

Comment:
Given that it's *possible* for a Dictionary to not have an `objId` property, this code could still be useful to avoid an infinite loop in the general case. Hence, please restore this code and the related unit-test!

Reply:
Yes I thought about that and it's why I added the `if (dict.objId)` above.
My assertion (tell me if I'm wrong) is that we can have no objId when the parent is in the object: ` << /Parent << /Parent << ....` but in such a case we can't have an infinite loop, so the only way to have an infinite loop is to have objId.

Reply:
And what happen if a pdf has an object with more than `LOOP_LIMIT` ancestors (which should be pretty rare I guess) ?

Reply:
> My assertion (tell me if I'm wrong) is that we can have no objId when the parent is in the object

Oh, you're probably correct here. The kind of structure required for an infinite loop to occur ought to be essentially impossible to create with only "direct" objects.

> And what happen if a pdf has an object with more than `LOOP_LIMIT` ancestors (which should be pretty rare I guess) ?

We simply broke out of the loop, since the intention was simply to avoid permanently hanging the worker-thread.